packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -19,8 +19,8 @@ import {
   getCommsSourcePath,
   getTimerWrapperSourcePath,
 } from '@agoric/swingset-vat';
-import { buildStorageInMemory } from '@agoric/swingset-vat/src/hostStorage';
-import buildCommand from '@agoric/swingset-vat/src/devices/command';
+import { buildStorageInMemory } from '@agoric/swingset-vat';
```

Comment:
Please move these two lines up into the above `import` (starting on line 12), just to group them visually.